R/class-workbook-wrappers.R

```diff
@@ -880,7 +879,7 @@ wb_remove_row_heights <- function(wb, sheet = current_sheet(), rows) {
 #' require(ggplot2)
 #' p1 <- ggplot(mtcars, aes(x = mpg, fill = as.factor(gear))) +
 #'   ggtitle("Distribution of Gas Mileage") +
-#'   geom_density(alpha = I(.5))
+#'   geom_density(alpha = 0.5)
```

Comment:
The `I()` function is not useful

Reply:
thanks, it's an old example covered over from `openxlsx`, though slightly updated for newer ggplot2 versions